src/plugins/dataflow/model/dataflow-content.ts

```diff
@@ -218,11 +217,6 @@ export const DataflowContentModel = TileContentModel
         const tableTileModel = getTileModelById(self, tableId); //get tableTile contents given a tableId
         const tableTileContent = tableTileModel?.content;
         self.sharedModel && sharedModelManager.removeTileSharedModel(tableTileContent, self.sharedModel);
-        //create a dataSet with two attributes with X / Y, link table tile to this dataSet
-        const title = tableTileModel?.title;
-        const newDataSet = createDefaultDataSet(title);
-        const newSharedDataSet = newDataSet && SharedDataSet.create({ providerId: tableId, dataSet: newDataSet });
-        sharedModelManager.addTileSharedModel(tableTileContent, newSharedDataSet);
```

Comment:
I believe this is still the desired behavior, but I'm not sure who's responsible for making it happen at this point.

Reply:
Perhaps it was added after your comment, but the `changeType === "unlink"` code above in the table-content seems like it should take care of this now.